admin-center/dashboard/site-overview.md

```diff
@@ -42,15 +42,15 @@ A list of the top 10 search terms with the greatest *gain* between yesterday and
 
 Use this report to identify newly popular terms. Create new content or update existing content to ensure it's current, accurate, and complete.
 
-## Queries with No Results
+## Queries With No Results
 
 A list of the top 10 queries that returned no results.
 
 Use this data to help searchers find your content by adding a [Best Bet]({{ site.baseurl }}/admin-center/content/best-bets.html), updating your existing web pages, or both.
 
-For example, on one agency's website, a dozen searches for *frostline* returned no results. The frost line&mdash;also known as depth of frost or freezing&mdash;is the depth to which the groundwater in soil is expected to freeze. The agency's pages on the topic referred to this concept as *depth of freezing*. 
+For example, on one agency's website, a dozen searches for *frostline* returned no results. The frost line &mdash; also known as depth of frost or freezing &mdash; is the depth to which the groundwater in soil is expected to freeze. The agency's pages on the topic referred to this concept as *depth of freezing*. 
 
-## Top Queries with Low Click Thrus
+## Top Queries With Low Click Thrus
```

Comment:
Were we dinged on consistency for "click through" vs. "click thru"?  I can't remember which we landed on

Reply:
No, but we should have been. I think we should go to "click through" and will make that edit throughout. Good catch.